### PR TITLE
Update a link in the autograd_tutorial.py

### PR DESCRIPTION
**Pull request by asrjy**
Updated link to momentum article on TDS

## Description
The link to the momentum's article now points to a different article. Updated with the medium article of the same content. 

## Checklist

- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

cc @svekars @sekyondaMeta @AlannaBurke @albanD @jbschlosser

### CODE REVIEW

File: beginner_source/blitz/autograd_tutorial.py [hunk 1]
```diff
@@ -67,7 +67,7 @@
 loss.backward() # backward pass
 
 ############################################################
-# Next, we load an optimizer, in this case SGD with a learning rate of 0.01 and `momentum <https://towardsdatascience.com/stochastic-gradient-descent-with-momentum-a84097641a5d>`__ of 0.9.
```
Comment (svekars) [inline beginner_source/blitz/autograd_tutorial.py:70]: this link still works for me

Comment (albanD) [reply beginner_source/blitz/autograd_tutorial.py:70]: The link works but redirects to a different article for some reason. The medium piece is the right one

**Review by albanD — APPROVED** [review]
